x/staging/posts/keeper/keeper_reactions.go

```diff
@@ -17,7 +17,7 @@ func (k Keeper) SavePostReaction(ctx sdk.Context, reaction types.PostReaction) e
 	key := types.PostReactionsStoreKey(reaction.PostID, reaction.Owner, reaction.ShortCode)
 	// Check for double reactions
 	if store.Has(key) {
-		return sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest,
+		return sdkerrors.Wrapf(types.ErrReactionAlreadyAdded,
```

Comment:
Can we use `DuplicatedRegisteredReaction` instead? This would make it coherent with other error names